src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+
+    public AuthenticationController(AuthenticationService service) {
+        this.service = service;
+    }
+
+    @PostMapping("/register")
+    public void registration(@RequestBody UserRequestDto requestDto) {
```

Comment:
```suggestion
    public void register(@RequestBody UserRequestDto requestDto) {
```

method names should start from verbs or be verbs